receipt_parser/importer.py

```diff
@@ -76,14 +76,22 @@ def rotate_image(input_file, output_file, angle=90):
     :return: void
         Rotates image and saves result
     """
+
+    if (angle == 0):
+        return None
+    elif (angle == -90):
+        angle = 90
+    elif (angle == 90):
+        angle = -90
+
     print(ORANGE + '\t~: ' + RESET + 'Rotate image' + RESET)
     with WandImage(filename=input_file) as img:
         with img.clone() as rotated:
             rotated.rotate(angle)
             rotated.save(filename=output_file)
 
 
-def sharpen_image(input_file, output_file):
+def sharpen_image(input_file, output_file, angle):
```

Comment:
I can see that you added `angle` to pass it to `rotate_image`, but I think we should rather move `rotate_image` out of `sharpen_image` and remove the `angle` parameter here. It's easier to test this way and again ensures separation of concerns.

Reply:
Updated this with last commit, thanks for your input! 👍